storage/persistent_storage/persistent_storage_local/persistent-storage-using-lvms.adoc

```diff
@@ -24,6 +24,24 @@ include::modules/lvms-installing-logical-volume-manager-operator-using-openshift
 
 include::modules/lvms-uninstalling-logical-volume-manager-operator-using-openshift-web-console.adoc[leveloffset=+2]
 
+include::modules/lvms-installing-logical-volume-manager-operator-disconnected-environment.adoc[leveloffset=+2]
+
+[role="_additional-resources"]
+.Additional resources
+
+* xref:../../../installing/disconnected_install/index.adoc[About disconnected installation mirroring]
```

Comment:
This technically works fine without an anchor/ID on docs.openshift.com, but has been known to cause problems on the Customer Portal build when missing. The ID for the top of this page would be:

```
* xref:../../../installing/disconnected_install/index.adoc#installing-mirroring-disconnected-about[About disconnected installation mirroring]
```